code/modules/mob/living/simple_animal/abnormality/zayin/pile_of_mail.dm

```diff
@@ -0,0 +1,253 @@
+//ripped off of wall gazer, sorry devs
+
+/mob/living/simple_animal/hostile/abnormality/mailpile
+	name = "Pile of mail"
```

Comment:
The abnormality is called "Letters on Standby", make sure to update all references of its name to reflect this.